src/azul/__init__.py

```diff
@@ -562,6 +562,14 @@ def github_project(self) -> str:
     def github_access_token(self) -> str:
         return os.environ['azul_github_access_token']
 
+    def portal_integrations_db_params(self):
```

Comment:
I'd split this up into two methods and make them properties.

I'd remove the encoding setting. I don't see why anyone would use anything but UTF-8. Thankfully, the times when we had to consider other encodings are behind us.